src/cuter_config.erl

```diff
@@ -0,0 +1,94 @@
+%% -*- erlang-indent-level: 2 -*-
+%%------------------------------------------------------------------------------
+-module(cuter_config).
+-behaviour(gen_server).
+
+%% External API.
+-export([fetch/1, store/2, start/0, stop/0]).
+%% gen_server callbacks
+-export([init/1, terminate/2, code_change/3, handle_info/2, handle_call/3, handle_cast/2]).
+
+-define(CONFIG_SERVER, config_server).
+
+-type from() :: {pid(), reference()}.
+-type name() :: atom().
+-type value() :: any().
+
+%% The persistent state of the server.
+-record(st, {
+  %% The storage for the configuration parameters.
+  parameters = orddict:new() :: orddict:orddict(name(), value())
+}).
+-type state() :: #st{}.
+
+%% ============================================================================
+%% External API
+%% ============================================================================
+
+%% Starts the configuration server.
+-spec start() -> ok.
+start() ->
+  case gen_server:start_link({local, ?CONFIG_SERVER}, ?MODULE, [], []) of
+    {ok, _Pid} -> ok;
+    {error, R} -> exit({config_start, R})
+  end.
+
+%% Stops the configuration server.
+-spec stop() -> ok.
+stop() ->
+  gen_server:call(?CONFIG_SERVER, stop).
+
+%% Stores the value for the given parameter.
+-spec store(name(), value()) -> ok.
+store(Parameter, Value) ->
+  gen_server:cast(?CONFIG_SERVER, {store, Parameter, Value}).
+
+%% Retrieves the value for the given parameter.
+-spec fetch(name()) -> {ok, value()} | enoent.
```

Comment:
I have two major comments here:
 1. You should make an appropriate type for the return type of `fetch/1` (currently `{ok, value()} | enoent`) and use that type also in `handle_call`, so that the connection is not lost.
 2. `enoent` is an error typically associated with a file/dir that does not exist.  Here it is used more like `not_found`, because AFAICS it is not returned just for files.  Perhaps `not_found` is better.

Reply:
Done.